packages/core/src/utils.test.ts

```diff
@@ -76,6 +88,6 @@ describe('maskStreamTags', () => {
   it('should handle whitespace around tags', async () => {
     const input = ['Before ', '  <secret>  ', 'hidden ', ' </secret>  ', ' after'];
     const masked = maskStreamTags(makeStream(input), 'secret');
-    expect(await collectStream(masked)).toBe('Before  after');
+    expect(await collectStream(masked)).toBe('Before      after');
```

Comment:
correct me if i'm wrong but i think this is the correct amount of space to expect here

Reply:
Yes, you're right! Discovered the same on another branch